src/display.c

```diff
@@ -27,9 +27,14 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <LCUI/config.h>
```

Comment:
config.h file is not public header file, you should use `#include "config.h"`

Reply:
I changed include path of `config.h` in d5cf404.